src/components/DynamicFormContainer/DynamicFormLayout.tsx

```diff
@@ -51,7 +51,8 @@ export const DynamicFormLayout: FunctionComponent = () => {
     const nextForms = [...forms];
     nextForms.splice(activeFormIndex, 1);
     setForms(nextForms);
-    const nextActiveFormIndex = nextForms.length > 0 ? nextForms.length - 1 : undefined;
+    const nextActiveFormIndex =
+      nextForms.length === activeFormIndex ? nextForms.length - 1 : nextForms.length > 0 ? activeFormIndex : undefined;
```

Comment:
I suggest the following

```ts
// if there is no forms left
if(nextForms.length === 0) setActiveFormIndex(undefined)
// if we delete the last form, then make active the previous form
else if (nextForms.length === activeFormIndex) setActiveFormIndex(nextForms.length - 1)
```
I think we should not allow removing the last form.

Reply:
Okay, i will disable the delete form button if there is last remaining form